src/guide/4-initial-conditions/5-from-disk.rst

```diff
@@ -1,4 +1,11 @@
 Initialisation from Disk
 ========================
 
-FLAMEGPU2 simulations can be initialised from disk using either the XML or JSON format.
\ No newline at end of file
+FLAMEGPU2 simulations can be initialised from disk using either the XML or JSON format. The format is described **TODO: Link to format**
```

Comment:
The XML format aims to be compatible with FLAMEGPU1, however contains additional information.